frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/redux/initial-state/vm-settings-tab-initial-state.ts

```diff
@@ -77,6 +77,7 @@ export const getInitialVmSettings = (data: CommonData): VMSettings => {
       isHidden: hiddenByProviderOrCloneCommonBaseDiskImage,
     },
     [VMSettingsField.CLONE_PVC_NAME]: {},
+    [VMSettingsField.DEFAULT_STORAGE_CLASS]: {},
```

Comment:
this would be better tied with the whole wizard, not just vm settings. Either directly in CommonData or as a CommonData reference